src/components/autocomplete/autocomplete.tsx

```diff
@@ -25,6 +25,7 @@ export interface AutocompleteProps {
     autoCompleteRef?: (api: AutocompleteApi) => any;
     filterSuggestions?: boolean;
     qeid?: string;
+    autoHighlight?: true; // ReactAutocomplete.Props['autoHighlight'] is missing the `true` default per https://github.com/argoproj/argo-ui/pull/560#issuecomment-2155872347
```

Comment:
oh defaults may not be supported for TS interface per the tests in my PR to DT: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69761#discussion_r1631989188

it's been a while since I've done more nuanced typings, my bad if so

Reply:
Yea

```tsx
<Autocomplete autoHighlight={false} />
```

has a typecheck error:
```
Type 'false' is not assignable to type 'true'.ts(2322
```

@Adrien-D would you mind PR'ing a fix to this by switching to JSDoc `@default`? I'd do it myself but then we'd need another reviewer with approval permissions 😅

Reply:
@agilgur5 Sure, [here is the fix ](https://github.com/argoproj/argo-ui/pull/561)

Reply:
Thanks!